assemblies/assembly-configuring-a-proxy.adoc

```diff
@@ -8,7 +12,8 @@ You can run the {product-very-short} application behind a corporate proxy by set
 
 Additionally, set the `NO_PROXY` environment variable to bypass the proxy for certain domains. The variable value is a comma-separated list of hostnames or IP addresses that can be accessed without the proxy, even if one is specified.
 
-include::modules/admin/procedure-understanding-no-proxy.adoc[leveloffset=+1]
```

Comment:
@themr0c Curious why this was removed? Is this intentional? This was added in #709 to clarify how the `NO_PROXY` env var was handled in RHDH, as a follow-up to a few customer issues.

Reply:
My mistake. Lost in rebase. Restoring it: https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/811